src/components/Support/Support.react.js

```diff
@@ -92,177 +97,167 @@ class Support extends Component {
       keyboardFocused={true}
       onTouchTap={this.handleClose}
     />;
+    const TopMenu = (props) => (
+      <div>
+        <div className="top-menu">
+          <FlatButton label="Overview" href="/overview" style={{ color: '#fff' }} className="topMenu-item" />
+          <FlatButton label="Docs" href="/docs" style={{ color: '#fff' }} className="topMenu-item" />
```

Comment:
@isuruAb Please redirect `Docs` to dev.susi.ai . Refer https://github.com/fossasia/chat.susi.ai/pull/595